scripts/qwik-city.ts

```diff
@@ -38,6 +38,9 @@ export async function buildQwikCity(config: BuildConfig) {
       './middleware/express': {
         import: './middleware/express/index.mjs',
         require: './middleware/express/index.cjs',
+      },
+        './middleware/netlify-edge': {
+          import: './middleware/netlify-edge/index.mjs',
```

Comment:
The formatting seems wrong
```suggestion
      './middleware/netlify-edge': {
        import: './middleware/netlify-edge/index.mjs',
```

Reply:
Good catch!